config.js

```diff
@@ -303,24 +303,31 @@ var config = {
     //     notifyAllParticipants: false
     // },
 
-    // Transcription (in interface_config,
-    // subtitles and buttons can be configured)
-    // transcribingEnabled: false,
-
-    // If true transcriber will use the application language.
-    // The application language is either explicitly set by participants in their settings or automatically
-    // detected based on the environment, e.g. if the app is opened in a chrome instance which is using french as its
-    // default language then transcriptions for that participant will be in french.
-    // Defaults to true.
-    // transcribeWithAppLanguage: true,
-
-    // Transcriber language. This settings will only work if "transcribeWithAppLanguage" is explicitly set to false.
-    // Available languages can be found in
-    // ./src/react/features/transcribing/transcriber-langs.json.
-    // preferredTranscribeLanguage: 'en-US',
-
-    // Enables automatic turning on captions when recording is started
-    // autoCaptionOnRecord: false,
+    // Transcription options (in interface_config,
```

Comment:
Drop the comment about interface_config. Also, please leave the old option names (without the explanation comments) and add a "DEPRECATED" title.